test/prototype_common_utils.py

```diff
@@ -54,7 +46,7 @@
 ]
 
 
-class PILImagePair(TensorLikePair):
+class ImagePair(TensorLikePair):
```

Comment:
I've refactored this to not handle mixed PIL / tensor image pairs. It was a problem if the tolerance is set for floating point images, i.e. in the range `[0.0, 1.0]`, but the comparison converted to uint8, which needs higher tolerances.